cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/logviewer/LogViewerWidget.kt

```diff
@@ -0,0 +1,126 @@
+package com.cognifide.cogboard.widget.type.logviewer
+
+import com.cognifide.cogboard.CogboardConstants.Props
+import com.cognifide.cogboard.config.service.BoardsConfigService
+import com.cognifide.cogboard.logStorage.LogStorage
+import com.cognifide.cogboard.storage.ContentRepository
+import com.cognifide.cogboard.widget.BaseWidget
+import com.cognifide.cogboard.widget.Widget
+import com.cognifide.cogboard.widget.connectionStrategy.ConnectionStrategy
+import com.cognifide.cogboard.widget.connectionStrategy.ConnectionStrategyFactory
+import com.cognifide.cogboard.widget.connectionStrategy.UnknownConnectionTypeException
+import com.cognifide.cogboard.widget.type.logviewer.logparser.LogParserStrategyFactory
+import io.vertx.core.Vertx
+import io.vertx.core.eventbus.MessageConsumer
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import main.kotlin.com.cognifide.cogboard.logStorage.LogStorageConfiguration
+import main.kotlin.com.cognifide.cogboard.logStorage.QuarantineRule
+
+class LogViewerWidget(
+    vertx: Vertx,
+    config: JsonObject,
+    serv: BoardsConfigService
+) : BaseWidget(vertx, config, serv) {
+    private val contentRepository: ContentRepository = ContentRepository.DEFAULT
+    private val address = config.endpointProp(Props.URL)
+    private var consumer: MessageConsumer<JsonObject>? = null
```

Comment:
Fix if possible: Could this be a `lateinit` variable?
https://kotlinlang.org/docs/properties.html#late-initialized-properties-and-variables

Reply:
Done.